samples/drivers/video/capture/src/main.c

```diff
@@ -125,10 +120,30 @@ int main(void)
 		return 0;
 	}
 
-	printk("- Default format: %c%c%c%c %ux%u\n", (char)fmt.pixelformat,
+	if (CONFIG_VIDEO_FORMAT_HEIGHT) {
```

Comment:
wouldn't it be to use `#if CONFIG_VIDEO_FORMAT_HEIGHT` instead of `if (CONFIG_VIDEO_FORMAT_HEIGHT)`?

Reply:
It was wondering the same thing: https://kernel.org/doc/html/latest/process/coding-style.html#conditional-compilation

Reply:
totally :)